paver.sh

```diff
@@ -98,7 +98,13 @@ clean_build() {
     fi
     # In some case pip hangs with a build folder in temp and
     # will not continue until it is manually removed.
-    rm -rf /tmp/pip*
+    # On the OSX build server tmp is in $TMPDIR
+    if [ ! -z "${TMPDIR-}" ]; then
```

Comment:
Two minor questions/suggestions:
  * why use `! -z` instead of `-n`?
  * why use parameter expansion when we are not interested in corner cases here?